pom.xml

```diff
@@ -91,6 +91,11 @@
       <artifactId>plain-credentials</artifactId>
       <version>1.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <version>4.5</version>
+    </dependency>
```

Comment:
I'm curious, don't we *also* have an httpclient somewhere else?

Reply:
Good call, my java-foo-less-ness led me to the first URIBuilder I found. Fixed.